.github/workflows/generate_artifacts.yml

```diff
@@ -0,0 +1,41 @@
+name: Generate Artifacts
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
should it be PR's as well to test how it it outputs prior to the merge?

Reply:
due to the noisy nature of the command making a bunch of requests to check if they 404 we want to limit it as much as possible, but can adjust moving forward.

Essentially the current setup is to generate every time on a merge to master, or direct push to master, and releases.  The other PR: https://github.com/bugcrowd/vulnerability-rating-taxonomy/pull/278 would instead make this a check whether or not it is up to date and require updating.